### ISSUE TITLE
training details

### ISSUE THREAD

**Issue by Castile** [lhaof/Motion-Guided-Attention#7]
Hi,nice work，sorry to bother you, I am very interest in your work.  would you share more training details for me? such as lr, bs, loss function & loss tricks，epoch or max iteration and some  other training tricks you think important, most of all， how is the training data organized, what augment have been made to the data set? My e-mail is zhl396740445@163.com. Looking forward your early reply!   THX!!!

**Comment by yujun0-0** [comment]
Hi,  I'm interested in your work too. Would you share the training details and the organization for training  dataset for me? My e-mail is yujunzhang@tju.edu.cn.  I'm looking forward your early reply!

**Comment by Edith-GHX** [comment]
hi, I'm interested in your work too. Would you share the train code? My e-mail is 616914002@qq.com. Looking forward your early reply,thanks.

**Comment by Lanezzz** [comment]
> hi, I'm interested in your work too. Would you share the train code? My e-mail is [616914002@qq.com](mailto:616914002@qq.com). Looking forward your early reply,thanks.

Do you have the interest of reproducing it together with me ? my QQ is 1164481749..

**Comment by Lanezzz** [comment]
> Hi, I'm interested in your work too. Would you share the training details and the organization for training dataset for me? My e-mail is [yujunzhang@tju.edu.cn](mailto:yujunzhang@tju.edu.cn). I'm looking forward your early reply!

Have you reproduced this paper? I hope someone can discuss VSOD with me .. my QQ is 1164481749
